BlueprintUI/Sources/Layout/LayoutMode.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+
+/// Controls the layout system that Blueprint uses to lay out elements.
+///
+/// Blueprint supports multiple layout systems. Each is expected to produce the same result, but
+/// some may have different performance profiles or special requirements.
+///
+/// You can change the layout system used by setting the ``BlueprintView/layoutMode`` property, but
+/// generally you should use the ``default`` option.
+///
+public enum LayoutMode: Equatable {
+    public static let `default`: Self = .legacy
```

Comment:
Do we expect to swap the default in time?

Reply:
Yep! We also need a way to change this at runtime, so it'll probably become a `var` before the feature branch lands.